### DISCUSSION TITLE
Losing Ibax Private Key Problem

### DISCUSSION THREAD

**Discussion by Explorerforrpcdot** [IBAX-io/go-ibax#1148] [Ideas] [answered]
Many people lost their private key when they switched to another computer or their phone broke.
----
It seems that these will cause many people to lose their ibax coin.
---
Emails can be added to ibax wallets for recovery purposes.

In this way, someone who forgets on another computer or whose computer is corrupted in some way can easily access his account again.

---
I hope it was useful. Don't forget to comment if you support.

**Comment by ibaxio** [comment, marked as answer]
Please refer to this discussion:  #62
